recipes-bsp/bootfiles/rpi-cmdline.bb

```diff
@@ -9,7 +9,9 @@ inherit deploy nopackages
 CMDLINE_DWC_OTG ?= "dwc_otg.lpm_enable=0"
 
 CMDLINE_ROOT_FSTYPE ?= "rootfstype=ext4"
-CMDLINE_ROOTFS ?= "root=/dev/mmcblk0p2 ${CMDLINE_ROOT_FSTYPE} rootwait"
+CMDLINE_ROOT_PARTITION ?= "${@oe.utils.conditional("ENABLE_USBBOOT", "1", "sda2", "mmcblk0p2", d)}"
```

Comment:
What about defining `CMDLINE_ROOT` defaulted to `/dev/mmcblk0p2`, which can be overwritten as required?

Reply:
Thank you for ther review. Yes, it is good. 
Is that means the revert the logic ?  
If ENABLE_USB = "0" , it will be set "mmcblk0p2" ? 
 or I should use another method to fix ? (like as sed ?)

Reply:
Not revert, but make it a variable.

Reply:
I got it. I understand your feedback, however, if user who want to overwrite the define , he should use two parameter...
I'll try it and please feedback again.